main.py

```diff
@@ -1,7 +1,48 @@
-import init_django_orm  # noqa: F401
+import init_django_orm # noqa:F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
-def main() -> QuerySet:
-    pass
+def main():
+    # Greate genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Drama")
+
+    # Create actors and actresses
+    Actor.objects.create(first_name="George",
+                         last_name="Klooney")
+    Actor.objects.create(first_name="Kianu",
+                         last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Keegan")
+    Actor.objects.create(first_name="Will",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Jaden",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Johansson")
+
+    # Update operations
+    (Genre.objects.filter(name="Dramma")
+     .update(name="Drama"))
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama' to match the created genre name and perform the update correctly.